sw/device/silicon_creator/lib/base/BUILD

```diff
@@ -72,8 +72,18 @@ cc_library(
         "//sw/device/silicon_creator/lib:keymgr_binding",
     ],
     # This library provides a special symbol that the linker will find.
-    # We want this to be a linker input to ensure the weak version in
-    # sec_mmio.c is overriden.
```

Comment:
nit: can you bring back this comment? and add it to epmp target as well? (with sec_mmio -> boot_measurements here and sec_mmio below)

Reply:
I deleted this comment because I couldn't find any weak symbol defined for `boot_measurements` just like I found for `sec_mmio_ctx`. Let me know if you still thing I should return the comment.

Reply:
> Should we at least make the declaration volatile? The idea is that rom_ext will consume the boot_measurements populated by rom.
> 
> ```c++
> volatile boot_measurements_t boot_measurements;
> ```

Done

Reply:
> 

We might not really need it but we can do a second pass after reaching a conclusion on this with @luismarques.

Reply:
> I deleted this comment because I couldn't find any weak symbol defined for `boot_measurements` just like I found for `sec_mmio_ctx`. Let me know if you still thing I should return the comment.

Ah, ok, makes sense. This discrepancy is because boot_measurements is data-only, i.e. it doesn't have any functions. I think it would be nice to clean this up so that everything is consistent. Specifically, every data structure in the `.static_critical` section has the following:
- A header file `name.h` that 
  - defines the data type
  - has an `extern` declaration
- A source file `name.c` that
  - has a weak definition
  - (optional) related functions
- A source file `static_critical_name.c` that 
  - provides a definition of the data structure and places in the `.static_critical.name` section.

IIRC `sec_mmio` has all these parts so we can use it as a template.